doc/user/USAGE.md

```diff
@@ -1,63 +1,130 @@
+| Drawing Exchange Format (DFX) | **.dxf** | Full |
+| DirectX | **.x** | Full | Yes |
+| 3D Manufacturing Format (3MF) | **.3mf** | Full |
+| Universal Scene Description | **.usd** | Full | Yes |
+| \***VDB** | **.vdb** | Default |
+
+\***Note:** As of version 2.5.0, F3D support for VDB is still experimental.
+
+## Constructing scenes
+
+For most file formats, F3D constructs **a default scene**, loading only the *geometry* from the file.
+
+However, formats supporting **full scenes** will automatically load additional scene properties including *lights*, *cameras*, *actors* and *texture*.
+
+If you'd like to disable these properties, you can specify the `--geometry-only` [option](OPTIONS.md) from the command line to construct a default scene instead.
+
+If you're unsure about what kind of scene your file supports, please see the table of [supported file formats](#supported-file-formats).
+
+## Interacting with your scene
+
+Once you've loaded your scene in F3D, you can interact with it by using your mouse and certain hotkeys. 
+
+### Moving the camera
+
+* *Click and drag* with the *left* mouse button to **rotate** around the focal point of the camera.
+* *Click and drag* vertically with the *right* mouse button <u>OR</u> *move the mouse wheel* to **zoom in/out**.
+* *Click and drag* with the *middle* mouse button to **translate** the camera.
+
+### Other shortcuts
+* Press <kbd>Enter</kbd> to reset the camera.
+* Press <kbd>Space</kbd> to play animation, if any.
+* Press <kbd>G</kbd> to toggle the horizontal grid.
+* Press <kbd>H</kbd> to display a cheatsheet of hotkey interactions. 
+
+For more information, see the [Interactions](INTERACTIONS.md) page.
+
+## Playing animations
+
+F3D can play [animations](ANIMATIONS.md) for any [supported files](#supported-file-formats) that contain them.
+
+With your file loaded into F3D, press the <kbd>W</kbd> hotkey to cycle through available animations. Then, press <kbd>Space</kbd> to play your selected animation. 
+
+Alternatively, you can use the command line to play animations. Use the `--animation-index` [option](OPTIONS.md) to specify which animation you want to play. To play all animations at once, use `--animation-index=-1` (.gltf/.glb only). 
+
+A few things to note about F3D's default animation settings:
+* F3D plays animations at a default speed factor of 1. Change the `--animation-speed-factor` [option](OPTIONS.md) to adjust how fast or slow your animation plays.
+* Additionally, F3D animates at a default frame rate of 60 frames per second. Change the `--animation-frame-rate` [option](OPTIONS.md) to adjust your desired frame rate.
+
+For more information, see the [Animations](ANIMATIONS.md) page.
+
+## Navigating plugins
+
+If you installed F3D using a package manager, it's possible that the packager chose to bundle the plugins in different packages or to list plugin dependencies as optional in order to reduce the reduce the number of dependencies of the main package.
```

Comment:
maybe add a paragraph before informing users that in most cases (eg, when using the binary release), plugins are loaded thanks for the configuration file and they do not need to worry about it.